SI/Sem.02/Pract.02/Task03.cpp

```diff
@@ -1,19 +1,20 @@
-#include <iostream>
-
-int main() {
-	double number1 = 0, number2 = 0;
-
-	std::cout << "Enter first number value: ";
-	std::cin >> number1;
-	std::cout << "Enter second number value: ";
-	std::cin >> number2;
+// ConsoleApplication1.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
 
-	if (number1 == number2) {
-		std::cout << "Both numbers are equal." << std::endl;
-		return 0;
-	}
+#include <iostream>
+using namespace std;
+int main()
+{
+	//�������� ��������, ����� ������ ��� �������� ����� � ����� ��� � �� ������.
```

Comment:
You are having problems with the encoding you use on the comments.
Are you writing comments in Cyrillic?